CHANGELOG.md

```diff
@@ -65,5 +65,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix double invocation of postCollection when MultiBucketCollector is present ([#14015](https://github.com/opensearch-project/OpenSearch/pull/14015))
 
 ### Security
+- Switch to iterative version of WKT format parser ([#14086](https://github.com/opensearch-project/OpenSearch/pull/14086))
```

Comment:
I wouldn't put this in the security section. It's really more of a simple bug fix to avoid excessive recursion. AFAIK, there's no security risk -- it's just preventing requests from failing on gnarly input.